OpenSourcePolitics/decidim-app
# [Feature] - Allow all locales

#### Description 

Allow system administrators to enable all locales in Decidim

- [x] Bump to latest revision
- [x] Update JS depencies 

### How to test

1. Create a new app and seed (seeds will take a moment because it generates translation for each available locale)
2. See available locales in app
3. Create a new organization and select some locales
4. See difference between the two organizations
